.github/workflows/test-add-compiler-matrix.yml

```diff
@@ -41,6 +41,8 @@ jobs:
     steps:
       - name: Checkout Source
         uses: actions/checkout@v4
+      - name: Install libcurl dev for gem deps
+        run: sudo apt update && sudo apt install -y libcurl4-openssl-dev
```

Comment:
In https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/743 was a different solution implemented as a workaround. It would be really great if all modules from Perforce use the same solution (and if it would be implemented in pdk-templates).

Reply:
Cheers, I've opened #528 to bring that fix into this module. Hopefully we don't need it forever🤞

Reply:
Thanks for pointing this out! I also opened https://github.com/puppetlabs/enterprise_tasks/pull/628 to fix this in enterprise_dist in the same way.

Reply:
I've opened an issue upstream to hopefully address the root cause: https://github.com/puppetlabs/orchestrator_client-ruby/issues/41

Reply:
Thanks for the updates; was out last week. I've rebased and dropped the curl patch and the test suite is passing now.